styles/base.css

```diff
@@ -1,18 +1,64 @@
 body {
-  padding: 1em;
+  line-height: 1.5;
+  font-size: 100%;
+  color: #404c47;
 }
 
 main {
-  margin: auto;
-  width: 80%;
+  margin: auto 0;
+  font-family: 'Roboto', sans-serif;
 }
 
-address {
-  display: inline-block;
+.site-header {
+  padding: 1em;
+  background-color: #43a0ad;
+  text-align: center;
+}
+
+.site-header img {
+  border-radius: 100%;
+  max-width: 150px;
+  width: 100%;
+}
```

Comment:
Thanks! 

    On Thursday, February 4, 2016 3:45 PM, Joel Stanner <notifications@github.com> wrote:
 

 In styles/base.css:>  }
>  
> -address {
> -  display: inline-block;
> +.site-header {
> +  padding: 1em;
> +  background-color: #43a0ad;
> +  text-align: center;
> +}
> +
> +.site-header img {
> +  border-radius: 100%;
> +  max-width: 150px;
> +  width: 100%;
> +}
Nice circle code.—
Reply to this email directly or view it on GitHub.